### ISSUE TITLE
Some text seems missing

### ISSUE THREAD

**Issue by weaverbel** [swcarpentry/lesson-example#200]
Just before Point 7, there is a sentence that just ends `and` with no further info or full stop and then it goes straight on to the point. Can this be right?

**Comment by rgaiacs** [comment]
> You can now click "Begin Import". When the process is done, you can click "Continue to repository" to visit your newly-created repository.
>
> Through the GitHub interface you can begin to edit and
>
> 7. If you want to work on the lesson from your local machine, you can now clone your newly-created repository to your computer:

I think this is right. But we can reword it to remove the "and".
